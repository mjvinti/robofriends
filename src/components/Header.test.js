import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

it('should render Header component', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
});

it('should handle shouldComponentUpdate', () => {
    const wrapper = shallow(<Header />);
    const shouldUpdate = wrapper.instance().shouldComponentUpdate({}, {});
    expect(shouldUpdate).toBeFalsy();
});