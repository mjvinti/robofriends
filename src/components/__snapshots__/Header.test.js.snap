// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component 1`] = `
<div>
  <h1
    className="f1"
  >
    RoboFriends
  </h1>
  <CounterButton
    color="red"
  />
</div>
`;
